db/schema.rb

```diff
@@ -353,6 +353,11 @@
     t.datetime "completed_at"
   end
 
+  create_table "user_profiles", force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8" do |t|
```

Comment:
This change should also update the migration version timestamp at the top of the file.